#! /bin/sh -e
## 02_paths.dpatch by Joerg Jaspert <joerg@debian.org>
## (Original made by some of the former Maintainers)
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjusts Path in manpage and adds silo definitions.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
	-patch) patch -f --no-backup-if-mismatch -p0 < $0;;
	-unpatch) patch -f --no-backup-if-mismatch -R -p0 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -urNad cdrecord/cdrecord.1 cdrecord/cdrecord.1
--- cdrecord/cdrecord.1	2004-02-29 18:13:20.000000000 +0100
+++ cdrecord/cdrecord.1	2004-03-04 21:12:56.000000000 +0100
@@ -157,7 +157,7 @@
 .I Scsibus 
 0 is the default SCSI bus on the machine. Watch the boot messages for more 
 information or look into 
-.B /var/adm/messages 
+.B /var/log/messages 
 for more information about the SCSI configuration of your machine.
 If you have problems to figure out what values for 
 .IR scsibus , target , lun
diff -urNad mkisofs/defaults.h cdrtools-2.0+a26pre27/mkisofs/defaults.h
--- mkisofs/defaults.h	2004-03-02 00:47:25.000000000 +0100
+++ mkisofs/defaults.h	2004-03-04 21:14:43.000000000 +0100
@@ -22,6 +22,9 @@
 #define	VOLUME_ID_DEFAULT	"CDROM"
 #define	BOOT_CATALOG_DEFAULT	"boot.catalog"
 #define	BOOT_IMAGE_DEFAULT	NULL
+#define  SILO_BOOT_IMAGE_DEFAULT "boot/second.b"
+#define  SILO_BOOTBLOCK_DEFAULT        "boot/cd.b"
+#define  SILO_CONF_FILE_DEFAULT        "/etc/silo.conf"
 #ifdef	APPLE_HYB
 #define	APPLE_TYPE_DEFAULT	"TEXT"
 #define	APPLE_CREATOR_DEFAULT	"unix"
